libs/go/libp2p_node/protocols/acn/v1_0_0/acn.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: acn
```

Comment:
Having a full spec and reusing it on the ACN has the benefit that we a) maintain consistency, b) document properly the protocol, c) make it easier for people to understand how it all fits together.

Reply:
It also makes it easier to write ACN peers in other languages